__tests__/commands/_helpers.js

```diff
@@ -17,6 +17,7 @@ const path = require('path');
 
 const installFixturesLoc = path.join(__dirname, '..', 'fixtures', 'install');
 
+// $FlowFixMe I don't understand the error
```

Comment:
me neither. cc @calebmer any idea what this is supposed to be?

<img width="847" alt="screen shot 2018-02-27 at 12 03 15" src="https://user-images.githubusercontent.com/13352/36727719-372e1074-1bb6-11e8-9593-872ec76ab0b4.png">